python/eos/analysis.py

```diff
@@ -282,10 +282,17 @@ def _perplexity(weights):
     def _ess(weights):
         """Helper function that computes the effective sample size of an array of weights"""
         return pypmc.tools.convergence.ess(weights)
+    
 
     def clone(self):
         """Returns an independent instance of eos.Analysis."""
         return eos.Analysis(**self.init_args)
+    
+    
+    def reset(self):
+        """Resets the analysis parameters to its initial state."""
```

Comment:
Can we call it reset_parameters and describe it as `Resets the analysis parameters to their default values.`?